db/version_edit.cc

```diff
@@ -43,6 +43,7 @@ enum Tag {
 enum CustomTag {
   kTerminate = 1,  // The end of customized fields
   kNeedCompaction = 2,
+  kPrivMeta = 10,
```

Comment:
The reason is that in some checking for saving log metadata (future patch) there is a collision with the tags themselves. Can we make the two sets mutually exclusive (e.g., start CustomTag at 300?) I can send a patch for that together with this one...